src/lib.rs

```diff
@@ -47,20 +47,20 @@ pub struct CoinSelectionOpt {
 
     /// The weight of the template transaction, including fixed fields and outputs.
     pub base_weight: u32,
-    /// Additional weight if we include the drain (change) output.
-    pub drain_weight: u32,
+    /// Additional weight if we include the change output.
+    pub change_weight: u32,
 
-    /// Weight of spending the drain (change) output in the future.
-    pub drain_cost: u64,
+    /// Weight of spending the change output in the future.
+    pub change_cost: u64,
 
     /// Estimate of cost of spending an input.
     pub cost_per_input: u64,
 
     /// Estimate of cost of spending the output.
     pub cost_per_output: u64,
 
-    /// Minimum value allowed for a drain (change) output.
-    pub min_drain_value: u64,
+    /// Minimum value allowed for a change output.
+    pub min_change_value: u64,
 
     /// Strategy to use the excess value other than fee and target
     pub excess_strategy: ExcessStrategy,
```

Comment:
#54 
Please replace `ToDrain` in excess strategy with `ToChange` 
```rust
pub enum ExcessStrategy {
    ToFee,
    ToRecipient,
    ToDrain,
}
```

Reply:
Done.